### ISSUE TITLE
CI failure for macos-latest, 8

### ISSUE THREAD

**Issue by milancurcic** [fortran-lang/stdlib#247]
CI is failing for macos-latest, 8 as of the last merge into master (#239). See https://github.com/fortran-lang/stdlib/runs/1438725329.

I have re-run the CI jobs to make sure the same error repeats. The error comes from a symbol missing in one of the shared libraries:

```
dyld: lazy symbol binding failed: Symbol not found: ___addtf3
  Referenced from: /Users/runner/work/stdlib/stdlib/build/src/tests/quadrature/test_trapz
  Expected in: /usr/lib/libSystem.B.dylib
```

The same error repeats for several other tests that rely on this symbol. Others are okay.

I have no knowledge of macOS so I'm not sure where to start with this one.

**Comment by certik** [comment]
Can you try to send a drFt PR reverting the last merge and see if it fixes the CI? It might not even be related to the last merge.

**Comment by Romendakil** [comment]
That  symbol should be part of libgcc_s.1.dylib, maybe a problem with the brewed gcc/gfortran-8?

**Comment by milancurcic** [comment]
@certik To confirm, I did what you suggested in #248.

**Comment by certik** [comment]
Thanks Milan. It looks like the error is indeed unrelated to the latest merge and rather is caused by some changes in the CI itself as Jurgen pointed out.

**Comment by 14NGiestas** [comment]
It was just a matter of updating the links as the warning said in the "Install Gfortran on MacOS" task.
I've fixed in #250

**Comment by milancurcic** [comment]
Fixed by #250.
